charts/azuremonitor-containers/templates/ama-logs-daemonset-windows.yaml

```diff
@@ -98,6 +98,9 @@ spec:
        - name: SIDECAR_SCRAPING_ENABLED
          value: {{ .Values.amalogs.sidecarscraping | quote }}
        volumeMounts:
+        - name: kube-api-access
+          mountPath: /var/run/secrets/kubernetes.io/serviceaccount
```

Comment:
Does this timebound token work for the scenario of the sev2? Can you add that in the testing scenarios you mentioned as well?

Reply:
> Does this timebound token work for the scenario of the sev2? Can you add that in the testing scenarios you mentioned as well?

> With the updated telegraf has the fix for refreshing the token. Yes, this works.  There is no special testing scenario and just applying the yaml to this cluster should restrict the token expiry for an hour and all the components should refresh the token before the expiry.